tests/rpmbuild.at

```diff
@@ -2272,3 +2272,44 @@ runroot rpmbuild \
 ],
 [ignore])
 AT_CLEANUP
+
+# ------------------------------
+# Check if dynamic spec generation works
+AT_SETUP([rpmbuild with dynamic spec generation])
+AT_KEYWORDS([build])
+RPMDB_INIT
+AT_CHECK([
+
+runroot rpmbuild -ba /data/SPECS/dynamic.spec
+],
+[0],
+[ignore],
+[ignore])
```

Comment:
I'd think the error message should be easy enough to catch here, especially if building with --quiet (as most of the test-suite does).

Reply:
Done.